setup.py

```diff
@@ -52,7 +52,8 @@ def main():
         author='Jeff Quast',
         author_email='contact@jeffquast.com',
         install_requires=('backports.functools-lru-cache>=1.2.1;'
-                          'python_version < "3.2"'),
+                          'python_version < "3.2"',
+                          'importlib_metadata;python_version < "3.8"'),
```

Comment:
This is only used for tests, right? 

But things in `install_requires` will be installed for all end users when pip installing.

Instead, can this be moved to `tox.ini` with the other test deps?

https://github.com/jquast/wcwidth/blob/4d4548048275ee25ba698cb0bd411296ff3f2238/tox.ini#L6-L7

Reply:
Good catch, let me sort that out.